src/constants/multicall/index.ts

```diff
@@ -1,13 +1,15 @@
 import { ChainId } from '@fuseio/fuse-swap-sdk'
 import MULTICALL_ABI from './abi.json'
+import { BINANCE_CHAIN_ID } from '../../connectors'
 
-const MULTICALL_NETWORKS: { [chainId in ChainId]: string } = {
+const MULTICALL_NETWORKS: any = {
```

Comment:
can you remind this what's that contract for?

Reply:
it aggregates multiple calls into a single call which is used in collecting the token balances

Reply:
ha, useful